modules/visual_script/visual_script.cpp

```diff
@@ -2042,11 +2056,14 @@ void VisualScriptInstance::create(const Ref<VisualScript> &p_script, Object *p_o
 
 	for (const Map<StringName, VisualScript::Variable>::Element *E = script->variables.front(); E; E = E->next()) {
 		variables[E->key()] = E->get().default_value;
-		//no hacer que todo exporte, solo las que queres!
+		//why equal to default value?? I think probably the default value is more than what it seems like
 	}
 
 	for (const Map<StringName, VisualScript::Function>::Element *E = script->functions.front(); E; E = E->next()) {
 
+		if(E->key() == "fn_312592")
```

Comment:
Huh? What is this for?

Reply:
This is the name I used for the default function(basically the graph itself) for now... 

I planned to change to something built-in inside of the Visual Script and use the get_graph or something from script to get the values but it works fine enough at the moment, and keeping the whole thing using the same function system is much easier to allow for maintaining compatibility.

Reply:
Technically it's a hack but using a weird name for default_func is actually a pretty ok solution. I need to move it to the Visual Script function itself. :P

So don't have to just copy paste strings.

Reply:
Are you able to create a separate list for the default_func? The scary part of this is if the other side can guess this function name, they're able to replace the entire graph.

Reply:
Are we trying to prevent hacking now?

But let them guess it, (it not human normal people don't use it) and even if they do it doesn't ever get executed. I will even remove it's last few fragments of it being added to the Godot Runtime.

Reply:
Are we trying to prevent hacking now?

But let them guess it, (it not human normal people don't use it) and even if they do it doesn't ever get executed. I will even remove it's last few fragments of it being added to the Godot Runtime.

Reply:
Are we trying to prevent hacking now?

But let them guess it, (it not human normal people don't use it) and even if they do it doesn't ever get executed. I will even remove it's last few fragments of it being added to the Godot Runtime.